gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/java/io/gravitee/plugin/entrypoint/webhook/configuration/WebhookEntrypointConnectorSubscriptionConfiguration.java

```diff
@@ -46,4 +47,10 @@ public String entrypointId() {
      * The list of headers to add to the request to the callback URL.
      */
     private List<HttpHeader> headers;
+
+    /**
+     * Ssl options.
+     */
+    @JsonProperty("ssl")
```

Comment:
Should the attribute be named ssl to avoid such annotation?

Reply:
:+1: